### ISSUE TITLE
Error on "Update Visible Elements" on some entries / Field conditions not working

### ISSUE THREAD

**Issue by superflausch** [spicywebau/craft-neo#726]
### Bug Description

We had some issues with missing field contents or wrong data within fields, which we think got addressed in other reports and bugfixes. 

But we still see errors in the consoles (browser as well as craft) as well as some field conditions won't apply correctly while editing (hide/show bases on other field). The error messages seem to match the issue:

```
yii\web\BadRequestHttpException: Request must accept JSON in response in /var/www/html/vendor/craftcms/cms/src/web/Controller.php:466
Stack trace:
#0 /var/www/html/vendor/spicyweb/craft-neo/src/controllers/Input.php(186): craft\web\Controller->requireAcceptsJson()
#1 [internal function]: benf\neo\controllers\Input->actionUpdateVisibleElements()
#2 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#4 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('update-visible-...', Array)
#5 /var/www/html/vendor/craftcms/cms/src/web/Application.php(302): yii\base\Module->runAction('neo/input/updat...', Array)
#6 /var/www/html/vendor/craftcms/cms/src/web/Application.php(627): craft\web\Application->runAction('neo/input/updat...', Array)
#7 /var/www/html/vendor/craftcms/cms/src/web/Application.php(281): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#8 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#9 /var/www/html/web/index.php(12): yii\base\Application->run()
#10 {main}

```
![Screenshot 2023-03-22 at 14 05 37](https://user-images.githubusercontent.com/539103/226913743-e6d616cf-a5e3-4e6d-83db-6c184a768ceb.png)

### Steps to reproduce

1. Visit entry detail page (with neo block field)
2. First errors already visible
3. Change some values of fields nested within neo
4. Additional errors get triggered

### Neo version

3.7.3

### Craft CMS version

4.4.5

### Does this issue involve templating, and if so, is eager-loading used?

This is not a templating issue

**Comment by ttempleton** [comment]
If possible, could you please send in your composer.json/lock and database backup to plugins@spicyweb.com.au, along with information about an entry that's being affected, and we'll have a look.

**Comment by superflausch** [comment]
@ttempleton Just sent out the email with the data. Thanks and let me know if you need additional info.

**Comment by ttempleton** [comment]
Thanks for sending that in @superflausch. I got a different error message, so I'm not certain that your issue will be resolved - please let me know if that error still occurs! But the error I got (`Call to a member function getFieldValue() on null`) was due to Neo failing to find the owner entry because the entry was disabled, and that's now fixed in Neo 3.7.4.
